transport/transport.go

```diff
@@ -63,13 +63,11 @@ type recvMsg struct {
 	err error
 }
 
-func (recvMsg) isItem() bool {
-	return true
-}
+func (*recvMsg) isItem() {}
 
 // All items in an out of a recvBuffer should be the same type.
 type item interface {
-	isItem() bool
```

Comment:
done

On Sat, Apr 2, 2016 at 12:06 AM, Xiang Li <notifications@github.com> wrote:

> In transport/transport.go
> <https://github.com/grpc/grpc-go/pull/615#discussion_r58288152>:
>
> >
> >  // All items in an out of a recvBuffer should be the same type.
> >  type item interface {
> > -	isItem() bool
>
> @tamird <https://github.com/tamird> If this is a marker method, I feel
> so. There are some code in stdlib using similar approach (
> https://golang.org/src/go/ast/ast.go: type Expr interface is an example)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc-go/pull/615/files/27c1c446d89d2e3f1e86424b9b02a79e7d8899f6#r58288152>
>